src/components/InventoryTable/EntityTableToolbar.test.js

```diff
@@ -842,18 +842,6 @@ describe('EntityTableToolbar', () => {
   });
 
   describe('export', () => {
-    const formats = ['json', 'csv'];
```

Comment:
At this level we don't need to be this specific I think. So this is moved and the hook itself is now tested with a random format.